de.stefan_oltmann.mines.yml

```diff
@@ -0,0 +1,52 @@
+app-id: de.stefan_oltmann.mines
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: Mines
+
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --device=dri
+  - --socket=x11
+
+modules:
+  - name: Mines
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 -t /app/share/applications de.stefan_oltmann.mines.desktop
+      - install -Dm 644 -t /app/share/metainfo/ de.stefan_oltmann.mines.metainfo.xml
+      - install -Dm 644 icon.png /app/share/icons/hicolor/128x128/apps/de.stefan_oltmann.mines.png
+      - install -Dm 644 icon_256.png /app/share/icons/hicolor/256x256/apps/de.stefan_oltmann.mines.png
+      - install -Dm 644 icon_512.png /app/share/icons/hicolor/512x512/apps/de.stefan_oltmann.mines.png
+      - mkdir -p /app/bin
+      - mkdir -p /app/lib
+      - cp -ru bin/* /app/bin/
+      - cp -ru lib/* /app/lib/
+    sources:
+      - type: archive
+        url: https://github.com/StefanOltmann/mines/releases/download/1.3.5/de.stefan_oltmann.mines-1.3.5.tar.gz
+        sha256: b8235a0c1ac96cc14fabf4e4004565e22d56b644a0f37dacfb077f63ac092535
+      - type: file
```

Comment:
The icon files are upstream, no reason whatsoever to have them here

https://github.com/StefanOltmann/mines/tree/main/icon